pkg/flow/grpc-logs.go

```diff
@@ -306,8 +307,22 @@ func (flow *flow) InstanceLogs(ctx context.Context, req *grpc.InstanceLogsReques
 	}
 
 	clients := flow.edb.Clients(ctx)
+	// its important to append the intanceID to the callpath since we don't do it when creating the database entry
+	prefix := cached.Instance.CallPath + "/instance:" + cached.Instance.ID.String()
```

Comment:
Manually building these strings seems to happen in a few places. At least `"/instance:"` does. This probably deserves to be its own function.